cfgov/jinja2/v1/the-bureau/bureau-structure/_macro-branches.html

```diff
@@ -0,0 +1,48 @@
+{# ==========================================================================
+
+   branches.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders an org chart branch when given:
+
+   categories: An Array of categories.
+
+   ========================================================================== #}
+{% macro render(categories) -%}
+{% import '_macro-node.html' as node with context %}
+<ul class="org-chart_branches
+           list
+           list__unstyled">
+    {% for category in categories %}
+    {% set category_index = loop.index %}
+    <li class="org-chart_branch {% if loop.last %} last-child {% endif %}">
+        <h4 class="org-chart_branch_name">{{ category.title }}</h4>
+        <ul class="org-chart_nodes
+                   nodes{{ category_index }}
+                   list
+                   list__unstyled">
+            <li class="org-chart_branch_name">
+                <button class="content-hide
+                               link_text
+                               jump-link
+                               jump-link__before
+                               jump-link__left">
+                    {{ category.title }}
+                </button>
+            </li>
+        {% for org in category.data %}
+            {{ node.render(org, category.sub_title ) }}
```

Comment:
Fixed in 9af73c9caf4d9b538b78f56a4bf061f0ad41f699. I just made the tagName a param.